package.json

```diff
@@ -134,7 +134,8 @@
     "body-parser": "1.20.3",
     "form-data": ">=4.0.4",
     "axios": "1.12.0",
-    "core-js": "3.45.1"
+    "core-js": "3.45.1",
+    "@babel/runtime-corejs2": "7.26.10"
```

Comment:
I noticed this changed in the lock files as a result of the exact versioning in CS3D. So I decided to lock it here too. However if it ever changes in CS3D it should change here too. This is where things get messy and challenging with versioning.

Reply:
I suppose I should say it gets more challenging when imports like this come from a third party we are not familiar with (i.e. unlike CS3D).